e2e/src/utils/typeoverride.js

```diff
@@ -241,6 +241,8 @@ const types = {
     }
 };
 
-const typeAlias = null;
+const typeAlias = {};
 
-module.exports = { types, typeAlias };
+const typeOverrides = {};
```

Comment:
What does the `typeOverrides` for?

Reply:
it's used on my debugging (try override some incorrect Polkadot.js types as we disscused on Discord), I think leave it here should covinient for future debugging

Reply:
OK